content/article/sabayon-is-back-and-kicking.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Sabayon is back - and kicking!"
+description = ""
+date = "2018-01-01"
+categories = [ "sabayon" ]
+tags = [
+"development",
+"sabayon"
+]
+author = "Ryuno-Ki"
++++
+
+**After more than a year of silence, we are back**
+
+We know, you were already wondering, whether this project is still alive,
+but rest assured that the development was steadily ongoing.
+
+A lot has changed, so we are excited to share the news with you! Let's go!
+
+<abbr title="Too long; didn't read">TL;DR</abbr>:
+* New team
+* New release strategy
+* New build system
+* Funding situation
+
+## New team
+
+In 2017 the team behind Sabayon grew.
+Now we have a test team, a webmaster and motivated "old dogs".
+So expect more changes to happen ;-)
+
+The focus of the webmaster will be on the website first, then turning to the
+wiki before looking on improving the forums.
+
+If you have ideas, feel free to drop them at
+ryuno-ki at sabayonlinux.org.
+
+## New release strategy
+
+Sabayon used to release a new version every month.
```

Comment:
Please, don't confuse our users even more, we already changed release strategy, and explained it [here under "Changes"](https://www.sabayon.org/release/latest-monthly-release-sabayon-1611/) .

Automatic (not always tested) monthly release will always be there, but we use to tag them manually into [stable/ directory](http://mirror3.mirror.garr.it/mirrors/sabayonlinux/stable/) when we decide to cut a new release.

Reply:
Addressed in https://github.com/Sabayon/sabayon-website-next/pull/63/commits/bf11b1a198e7f20591d3e0ec5fb76cbe9b5e5505